### ISSUE TITLE
Do something and Copy the Generated Link to Clipboard

### ISSUE THREAD

**Issue by nootanghimire** [Avasz/lmgtfy-js#2]
It would be a nice feature to copy the generated link to clipboard for further use

**Comment by Avasz** [comment]
yeah, was thinking of it before writing the addon, but doing some research in stackoverflow, the experts were saying it is not safe for client side language to be copying stuffs to machine's clipboard.
But I will try. :dancer:
